src/Command/Db/DbSizeCommand.php

```diff
@@ -287,8 +318,18 @@ private function mysqlNonInnodbQuery(array $database, $excludeInnoDb = true)
      */
     private function mysqlInnodbQuery(array $database)
     {
-        $query = 'SELECT SUM(ALLOCATED_SIZE) FROM information_schema.innodb_sys_tablespaces;';
-
+        $deployment = $this->api()->getCurrentDeployment($this->getSelectedEnvironment());
+        $service    = $deployment->getService($database['service']);
+        $databases  = array_keys($service->configuration['endpoints'][$database['rel']]['privileges']);
+        
+        //convert database names to OR statements that the sys_tablespaces can use.
+        $strDbs     = implode(' OR ', array_map(
+                                        function($database) {
+                                            return "`NAME` LIKE \"$database/%\"";
```

Comment:
It comes from the service configuration so it's probably OK, but it would be safer to escape the string for SQL here

Reply:
I can't use https://www.php.net/manual/en/mysqli.real-escape-string.php since we don't have a connection object.
Is addslashes sufficient?

Reply:
I added addslashes